_includes/coffee/main.coffee

```diff
@@ -81,3 +81,26 @@ link_html = (link, data) ->
   #{data}
   </a>
   """
+
+github_to_raw = (data) ->
+  ### Convert a GitHub URL to raw format
+
+  Args:
+    data: the GitHub URL
+
+  Returns:
+    the updated url
+  ###
+  pattern = ->
+    ///
+    https:\/\/github\.com\/([A-Za-z0-9_-]{3,15})\/([A-Za-z0-9_.-]+)\/blob\/(.+)$
```

Comment:
What if someone
- types `http://`
- types `www.github.com`
- types `GitHub.com`
- has a username longer than 15 chars ([max. is 39](https://github.com/shinnn/github-username-regex))

Consider revising the pattern to be more forgiving:
`[Gg]it[Hh]ub\.com\/([A-Za-z0-9_-]{3,39})\/([A-Za-z0-9_.-]+)\/blob\/(.+)$`